licenserc.toml

```diff
@@ -0,0 +1,24 @@
+# Base directory for the whole execution.
+baseDir = "."
+
+# Path to the header template file.
+# The header template file is skipped on any execution.
+headerPath = "license-header.txt"
+
+# On enabled, check the license header matches exactly with whitespace.
+strictCheck = true
+
+# Patterns of path to be included on execution.
+includes = ["**/*.rs", "**/*.go"]
+
+# Whether you use the default mapping.
+useDefaultMapping = true
+
+additionalHeaders = ["license-style.txt"]
+
+[mapping.GO_STYLE]
+extensions = ["go"]
+
+# Properties to fulfill the template.
+[properties]
+Year = 2024
```

Comment:
What's the point of this `Year` variable? It's not like we ever use anything other than the current year so wondering if we can just hard-code it in as part of the license instead.